src/index.tsx

```diff
@@ -4,12 +4,15 @@ import "./index.css";
 import { App } from "./app";
 import reportWebVitals from "./reportWebVitals";
 
-const root = ReactDOM.createRoot(document.getElementById("root"));
-root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
-);
+const el = document.getElementById("root");
```

Comment:
Typescript was complaining here that  `document.getElementById("root");` could be empty which would make the below code fail. I check below that it isn't a falsy value so I know it can render.